ui/src/data-products/element-list/element-table-item.tsx

```diff
@@ -60,22 +61,50 @@ export const ElementTableItem: React.FC<ElementTableItemProps> = ({
                 {showLocationTrackName && <td>{locationTrackName}</td>}
                 <td>{geometryAlignmentName}</td>
                 <td>{type}</td>
-                <td>{trackAddressStart && formatTrackMeter(trackAddressStart)}</td>
-                <td>{trackAddressEnd && formatTrackMeter(trackAddressEnd)}</td>
+                <td className={styles['element-list-view__column--number']}>
+                    {trackAddressStart && formatTrackMeter(trackAddressStart)}
+                </td>
+                <td className={styles['element-list-view__column--number']}>
+                    {trackAddressEnd && formatTrackMeter(trackAddressEnd)}
+                </td>
                 <td>
                     <CoordinateSystemView coordinateSystem={coordinateSystem} />
                 </td>
-                <td>{roundToPrecision(locationStartE, Precision.TM35FIN)}</td>
-                <td>{roundToPrecision(locationStartN, Precision.TM35FIN)}</td>
-                <td>{roundToPrecision(locationEndE, Precision.TM35FIN)}</td>
-                <td>{roundToPrecision(locationEndN, Precision.TM35FIN)}</td>
-                <td>{roundToPrecision(length, Precision.measurementMeterDistance)}</td>
-                <td>{curveRadiusStart}</td>
-                <td>{curveRadiusEnd}</td>
-                <td>{cantStart && roundToPrecision(cantStart, Precision.cantMillimeters)}</td>
-                <td>{cantEnd && roundToPrecision(cantEnd, Precision.cantMillimeters)}</td>
-                <td>{angleStart}</td>
-                <td>{angleEnd}</td>
+                <td className={styles['element-list-view__column--number']}>
+                    {roundToPrecision(locationStartE, Precision.TM35FIN)}
+                </td>
+                <td className={styles['element-list-view__column--number']}>
+                    {roundToPrecision(locationStartN, Precision.TM35FIN)}
+                </td>
+                <td className={styles['element-list-view__column--number']}>
+                    {roundToPrecision(locationEndE, Precision.TM35FIN)}
+                </td>
+                <td className={styles['element-list-view__column--number']}>
+                    {roundToPrecision(locationEndN, Precision.TM35FIN)}
+                </td>
+                <td className={styles['element-list-view__column--number']}>
+                    {roundToPrecision(length, Precision.measurementMeterDistance)}
+                </td>
+                <td className={styles['element-list-view__column--number']}>
+                    {curveRadiusStart != undefined &&
```

Comment:
Näissä tarkoituksella `!=` (catchaa sekä undefinedin että mahdolliset nullit.) Olisin halunnut muuttaa näitä tässä yhteydessä vaan suoraan undefinedeiksi, mutta nää taisivat tulla bäkkäriltä ihan nulleina silloin kun ne nulleja ovat, eikä frontti muuta niitä undefinedeiksi käsin